JavaScript/function.sublime-snippet

```diff
@@ -1,5 +1,7 @@
 <snippet>
-    <content><![CDATA[function($1) {${0:$TM_SELECTED_TEXT}};]]></content>
+    <content><![CDATA[function($1) {
+	${0:$TM_SELECTED_TEXT}
+}]]></content>
```

Comment:
I don't think it's a good idea to always make this multi-line. You can easily turn the single-line anonymous function into a multi-line one by pressing enter/return once, but not the other way around.